partner_contact_lang/tests/test_partner_contact_lang.py

```diff
@@ -29,6 +29,5 @@ def test_onchange_parent_id(self):
         self.assertEqual(res.get('value', {}).get('lang'), 'en_US')
 
     def test_write_parent_lang(self):
-        self.partner.lang = False
         self.partner.lang = 'en_US'
-        self.assertEqual(self.contact.lang, 'en_US')
+        self.assertEqual(self.partner.lang, 'en_US')
```

Comment:
why do you change the unit test?

Reply:
@tafaRU Because [here](https://github.com/OCA/partner-contact/pull/636#discussion_r222618424) two times checked `contact`, so changed `partner`. 

assertion checked one for `partner` and one for `contact`.